src/three-components/CachingGLTFLoader.ts

```diff
@@ -150,6 +153,18 @@ export class CachingGLTFLoader {
         if (!node.name) {
           node.name = node.uuid;
         }
+        if (!(node as Mesh).isMesh) {
+          return;
+        }
+        const mesh = node as Mesh;
+        const materials =
+            Array.isArray(mesh.material) ? mesh.material : [mesh.material];
+        materials.forEach(material => {
+          if ((material as any).isMeshStandardMaterial) {
+            this.roughnessMipmapper.generateMipmaps(
+                material as MeshStandardMaterial);
```

Comment:
I've done a little work to check this. The actual render calls are very fast, all together only a few ms. It looks like it takes a long time, but that's just because it makes it upload the textures, which otherwise happens on the first animation frame. Then there's that DPR bug, where on a macbook it takes a long time to setDPR, so I intend to fix that.

Reply:
Can we file an issue the DPR hot path? Otherwise SGTM

Reply:
#886